mlrun/model_monitoring/db/tsdb/base.py

```diff
@@ -446,3 +446,26 @@ def df_to_results_values(
             )
 
         return metrics_values
+
+    @staticmethod
+    def _get_start_end(
+        start: Union[str, datetime],
+        end: Union[str, datetime],
```

Comment:
just to keep consistency with the syntax of other functions in this class. I’ve noticed that this issue also appears in other parts of the class, so please update those as well. Afterward, you can remove the `from typing import Union` statement.
```suggestion
        start: typing.Union[str, datetime],
        end: typing.Union[str, datetime],
```